### PR TITLE
chore: add dev-dependency disclaimer

### PR DESCRIPTION
**Pull request by mathur07**
SS
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,4 +128,7 @@ Apache 2.0, See [LICENSE](LICENSE) for more information.
 
 # Data and telemetry
 
-The Red Hat Dependency Analytics Extension for Visual Studio Code collects anonymous [usage data](Telemetry.md) and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.elemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-commons#how-to-disable-telemetry-reporting
\ No newline at end of file
+The Red Hat Dependency Analytics Extension for Visual Studio Code collects anonymous [usage data](Telemetry.md) and sends it to Red Hat servers to help improve our products and services. Read our [privacy statement](https://developers.redhat.com/article/tool-data-collection) to learn more. This extension respects the `redhat.elemetry.enabled` setting which you can learn more about at https://github.com/redhat-developer/vscode-commons#how-to-disable-telemetry-reporting
+
+
+> **Disclaimer** : Dependency Analytics does not analyse dev/test dependencies associated with the Stack.
```
Comment (arajkumar) [inline README.md:134]: IMHO, this should be moved to the top, probably after the short intro about the extension.

@prashbnair WDYT?

**Comment by maheshwari-git** [comment]
lgtm

File: README.md (hunk 1 above)
Comment (prashbnair) [reply README.md]: Yes, it should be placed on top.

Comment (mathur07) [reply README.md]: ack

**Comment by mathur07** [comment]
@arajkumar PTAL

File: README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Dependency Analytics is powered by [Snyk Intel Vulnerability DB](https://snyk.io
 - Suggests a project level license, check for conflicts between dependency licences
 - AI based guidance for additional, alternative dependencies
 
+> **Disclaimer** : Dependency Analytics does not analyse dev/test dependencies associated with the Stack.
```
Comment (arajkumar) [inline README.md:16]: How about this?
```suggestion
> :warning: Dependency Analytics does not analyse **dev/test** dependencies.
```

Comment (prashbnair) [inline README.md:16]: **Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies associated with the Stack.

Comment (prashbnair) [inline README.md:16]:
```suggestion
> **Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies associated with the Stack.
```

Comment (prashbnair) [reply README.md:16]: I feel disclaimer is required. Also dev/test needs to be highlighted.

Comment (mathur07) [reply README.md]: ack

Comment (mathur07) [reply README.md]: ack

**Review by arajkumar — APPROVED** [review]
lgtm, wait for @prashbnair 's approval.

File: README.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ Dependency Analytics is powered by [Snyk Intel Vulnerability DB](https://snyk.io
 - Suggests a project level license, check for conflicts between dependency licences
 - AI based guidance for additional, alternative dependencies
 
+ > :warning: **Disclaimer**: Dependency Analytics does not analyse **dev/test** dependencies.
```
Comment (prashbnair) [inline README.md:16]: Do we need warning and disclaimer, just disclaimer should be good enough.

Comment (mathur07) [reply README.md:16]: okay, let's remove the warning then.

**Pull request merged**